kanopi/kanopi-pack-asset-loader
# Adjust asset loader for multiple asset revisions per entry

## Description
> As a developer, I need to make asset loader changes to accommodate changes to the asset manifest from Kanopi Pack

## Steps to Validate
1. Update the package
2. Update Kanopi Pack to v2.x
3. Make some file changes to ensure some of the entry points have updates
4. Confirm at the /webpack-dev-server that the effected entry point lists .hot-update.js/css
5. Load the site using the Asset Loader and confirm assets load
